test/test.cpp

```diff
@@ -520,11 +522,11 @@ void testEvent() {
 
 void testQuantity() {
 	cout<<"Quantity test: ";
-	Quantity t1(Units::Time);
+	Quantity t1("2days",Units::Time);
 	Quantity t2(1, "h", Units::Time);
 	Quantity t3("60min", Units::Time);
 	Quantity t4("3601 sec", Units::Time);
-	t1.fromString("2days");
+	//t1.fromString("2days");
```

Comment:
Is commenting this out intended? Without this we are no longer testing anywhere the `fromString` method